.evergreen/config.in.yml

```diff
@@ -94,6 +94,58 @@ functions:
           - .evergreen/run-kms-servers.sh
         env:
           DRIVERS_TOOLS: ${DRIVERS_TOOLS}
+
+  "bootstrap oidc":
```

Comment:
I tried to do these all as subprocess.exec with scripts but both EVG and AWS were not forgiving and I ended up with 1 of 3 instead of all 3. The Python reference PR uses shell.exec for all 3.